src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/orchestrator/network/_network_orchestrator.py

```diff
@@ -0,0 +1,37 @@
+from ...orchestrator._orchestrator import Orchestrator
+import ipaddress
+from avs.avsarconboarder.exception import InvalidInputError
+
+class NetworkOrchestrator(Orchestrator):
+    def __init__(self):
+        self.list_of_ip_addr_in_segment = list()
+
+    def validate_static_ip_cidr_block(self, config):
+        segmant_ip_cidr = config["staticIpNetworkDetails"]["networkCIDRForApplianceVM"]
+        gateway_ip      = ""
+        char_index      = 0
+        while char_index < len(segmant_ip_cidr) and segmant_ip_cidr[char_index] != '/':
+            gateway_ip += segmant_ip_cidr[char_index]
+            char_index += 1
+
+        cidr = segmant_ip_cidr[char_index - len(segmant_ip_cidr) + 1:]
+        if int(cidr) != 28:
```

Comment:
Aren't we supposed to support block sizes > /28 as well?

Reply:
We have decided to use /28 size because of following reasons:
1. not much ambiguity to the customer.
2. customer would not like to waste his address space unnecessarily by giving a bigger address space.
3. even in other Azure RP like VNET creation. It is mentioned to provide a /24 or /22 size explicitly, I feel customer might not feel an issue if we are going to restrict.

Let's keep this convo open till we get a conclusion on this.

Reply:
We have agreed on /28 cidr